packages/node/package.json

```diff
@@ -7,7 +7,7 @@
   "license": "MIT",
   "scripts": {
     "build": "rollup -c",
-    "test": "jest --setupFiles dotenv-extended/config --runInBand --bail --forceExit",
+    "test": "jest --setupFiles dotenv-extended/config --runInBand --bail --forceExit --verbose",
```

Comment:
Typically these options get set in `jest.config.json`

Reply:
including `--runInBand`, etc?

Reply:
`--runInBand` is an exception https://github.com/facebook/jest/issues/3215